Exercise_C/index.html

```diff
@@ -3,14 +3,27 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title></title>
+    <title>Exercise C</title>
     <meta name="description" content="" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
     <!-- CODE HERE -->
 
+
+    <div class="box"> 
+
+    <div class="blue">  </div>
+
+    <div class="green">  </div>
+    </div>
+
+
+    <div class="redBox">     <div class="red">  </div>
```

Comment:
For this, if you would like to add two classes
You would need to change this to `div class="redBox red"` this would allow you to add two classes